frontend/express/public/javascripts/countly/countly.template.js

```diff
@@ -579,6 +579,35 @@ $.expr[":"].contains = $.expr.createPseudo(function(arg) {
     };
 });
 
+/**
+* Set menu items by restriction status, hiding empty menu-categories
+* @name setMenuItems
+* @global
+*/
+function setMenuItems() {
+    // hide empty section headers for types listed below
+    var types = ['web', 'mobile', 'desktop'], categories;
```

Comment:
We cannot hardcode this. Types can be dynamically added/removed/absent etc
I think it would be enough if it would work on the currently selected app type?
Or if not, you can use it on `this.appTypes`